frontend/components/FleetAce/FleetAce.tsx

```diff
@@ -12,6 +12,7 @@ import "./mode";
 import "./theme";
 
 export interface IFleetAceProps {
+  focus?: boolean;
```

Comment:
Would something like `autofocusEnabled` be a more clear prop name?

Reply:
I just matched whatever the final components autofocus key (we have a `focus` on `react-ace`'s `AceEditor`, a `autofocus` on our component `InputField` , and a `autoFocus` on `react-select`'s `Select` for the final components).

Let me know if you want to standardize it to match `autofocus` used on our component `InputField`

Reply:
I think it's fine now as-is. That does highlight our need to standardize prop names at some point, but not a priority right now.